app/serializers/serialize_exercise_assignment.rb

```diff
@@ -32,7 +32,7 @@ def overview
   end
 
   def general_hints
-    hints["general"].to_a
+    hints[0].to_a
```

Comment:
Is `general` always first? Is that in the spec?

Reply:
No, but the first hint starts at `1` and if there is no number there (meaning: "General") the `to_i` call resolves to `0` :)

Reply:
#rubyskillz